lib/opentok/token_generator.rb

```diff
@@ -81,6 +81,15 @@ def generate_token
         end
         data_params[:connection_data] = data
       end
+
+      if opts[:initial_layout_classes]
```

Comment:
@aiham, @lucas, I think the name of the param in Ruby should be `initial_layout_class_list` (not `initial_layout_classes` or `layoutClassList`).